src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -684,7 +887,7 @@ tests:
       response:
           values:
               - name: "Status"
-                value: 0x00
+                value: 0x00 || 0x89
```

Comment:
I don't think this does the right thing either.  It generates code like this:
```
VerifyOrReturn(CheckValue("status", value.status, static_cast<uint8_t>(0x00 || 0x89)));
```

Reply:
I think for now you should probably assume you get a status of 0 and file a followup to improve this once the harness can handle it.

That's assuming all current devices would require enough scenes that the test would pass.  Would they require that?